doc/man3/OSSL_PARAM_BLD.pod

```diff
@@ -141,12 +141,12 @@ private key.
     OSSL_PARAM *params;
```

Comment:
May as well set this to NULL which you are here...
````
OSSL_PARAM *params;
````

Same in the next example also.